src/client/app/translations/data.js

```diff
@@ -297,6 +297,8 @@ const localeData = {
 		"map.is.not.calibrated": "Calibration Needed",
 		"map.is.not.displayable": "Display Disabled",
 		"map.load.complete": "Map load complete from",
+		"map.bad.load": "Map image needed",
```

Comment:
This should be placed alphabetically with the other map.bad items. There also needs to be translations in the other two languages in this file as with other keys.

Also could the string become "Map image file needed".